<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btLoadImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CargarVolumen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSliceNum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice:</string>
    </property>
   </widget>
   <widget class="QSlider" name="slSliceNumber">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAplyEffect">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chUseImageProcessed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usar Imagen Destacada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Efectos Vision:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbImageBrats">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtVideoImages">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btGenerateVideo">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>100</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSaveImage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSliceImageProcessed">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin volumen cargado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSliceImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>Sin volumen cargado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione una Imagen nii</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
